odoo/models.py

```diff
@@ -3853,7 +3853,7 @@ def _parent_store_update(self):
         if prefix:
             parent_ids = {int(label) for label in prefix.split('/')[:-1]}
             if not parent_ids.isdisjoint(self._ids):
-                raise UserError(_("Recursivity Detected."))
+                raise UserError(_("Recursion Detected."))
```

Comment:
unique case of translation affected

Reply:
Only this can be merged as said by @mart-e  in #36984. Please make only a commit for this user-visible typo ;) and not the code comments typos.

Reply:
done! @mart-e @Feyensv